x/ccv/consumer/module.go

```diff
@@ -158,8 +158,9 @@ func (am AppModule) BeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) {
 		// the CCV channel was established, but it was then closed;
 		// the consumer chain is no longer safe
 
-		// cleanup state
+		// clean up states
 		am.keeper.DeleteProviderChannel(ctx)
+		am.keeper.DeleteProviderClient(ctx)
```

Comment:
What about the rest of the state? i.e., see setters, e.g., `SetPort`, `SetUnbondingTime`, `SetPendingChanges`, `SetPacketMaturityTime`, `SetHeightValsetUpdateID`, `SetOutstandingDowntime`, `SetCCValidator`, `SetPendingSlashRequests`

Reply:
Seems like almost all the state created by the module should be deleted really